geomesa-tools/bin/geomesa-common.sh

```diff
@@ -0,0 +1,174 @@
+        read -p "Is this intentional? Y\n " -n 1 -r
+        if [[  $REPLY =~ ^[Nn]$ ]]; then
+            GEOMESA_LIB=${GEOMESA_HOME}/lib
+            echo >&2 ""
+            echo "Now set to ${GEOMESA_LIB}"
+        fi
+        echo >&2 ""
+    fi
+
+    echo >&2 ""
+    echo "To persist the configuration please update your bashrc file to include: "
+    echo "export GEOMESA_HOME="$GEOMESA_HOME""
+    echo "export PATH=\${GEOMESA_HOME}/bin:\$PATH"
+}
+
+GEOMESA_OPTS="-Duser.timezone=UTC -DEPSG-HSQL.directory=/tmp/$(whoami)"
+GEOMESA_OPTS="${GEOMESA_OPTS} -Djava.awt.headless=true"
+GEOMESA_DEBUG_OPTS="-Xmx8192m -XX:MaxPermSize=512m -XX:-UseGCOverheadLimit -Xdebug -Xnoagent -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=9898"
+GEOMESA_CP=""
+
+# Define GEOMESA_HOME and update the PATH if necessary.
+if [ -z "$GEOMESA_HOME" ]; then
+setGeoHome
+else
+    echo >&2 "Using GEOMESA_HOME = $GEOMESA_HOME"
+    if [[ $1 = configure ]]; then
+        read -p "Do you want to reset this? Y\n " -n 1 -r
+        if [[  $REPLY =~ ^[Yy]$ ]]; then
+            echo >&2 ""
+            setGeoHome
+        fi
+        echo >&2 ""
+    fi
+fi
+
+# GEOMESA paths, GEOMESA_LIB should live inside GEOMESA_HOME, but can be pointed elsewhere in geomesa-env
+if [ -z "$GEOMESA_LIB" ]; then
+    GEOMESA_LIB=${GEOMESA_HOME}/lib
+elif [[ $1 = configure ]]; then
+    message="Warning: GEOMESA_LIB was already set, probably by a prior configuration or the geomesa-env config."
+    message="${message}\n The current value is ${GEOMESA_LIB}."
+    echo >&2 ""
+    echo -e >&2 "$message"
+    echo >&2 ""
+    read -p "Do you want to reset this to ${GEOMESA_HOME}/lib? Y\n " -n 1 -r
+    if [[  $REPLY =~ ^[Yy]$ ]]; then
+        GEOMESA_LIB=${GEOMESA_HOME}/lib
+        echo >&2 ""
+        echo "Now set to ${GEOMESA_LIB}"
+    fi
+    echo >&2 ""
+fi
+
+# Configure geomesa conf directory this can be set in geomesa-env
+if [ -z "$GEOMESA_CONF_DIR" ]; then
+    GEOMESA_CONF_DIR=${GEOMESA_HOME}/conf
+elif [[ $1 = configure ]]; then
+    message="Warning: GEOMESA_CONF_DIR was already set, probably by a prior configuration or the geomesa-env config."
+    message="${message}\n The current value is ${GEOMESA_CONF_DIR}."
+    echo >&2 ""
+    echo -e >&2 "$message"
+    echo >&2 ""
+    read -p "Do you want to reset this to ${GEOMESA_HOME}/conf? Y\n " -n 1 -r
+    if [[  $REPLY =~ ^[Yy]$ ]]; then
+        GEOMESA_CONF_DIR=${GEOMESA_HOME}/conf
+        echo >&2 ""
+        echo "Now set to ${GEOMESA_CONF_DIR}"
+    fi
+    echo >&2 ""
+fi
+
+GEOMESA_OPTS="${GEOMESA_OPTS} -Dlog4j.configuration=file://${GEOMESA_CONF_DIR}/log4j.properties"
+
+# Configure geomesa logging directory this can be set in geomesa-env
+if [[ -z "${GEOMESA_LOG_DIR}" ]]; then
+  export GEOMESA_LOG_DIR="${GEOMESA_HOME}/logs"
+  export GEOMESA_OPTS="-Dgeomesa.log.dir=${GEOMESA_LOG_DIR} $GEOMESA_OPTS"
+fi
+
+# Configure Java Library Path this can be set in geomesa-env.
+if [ -n "$JAVA_LIBRARY_PATH" ]; then
```

Comment:
Should I just standardize that across all these files then? Converting the appropriate logic as well?